manifests/config/passenger.pp

```diff
@@ -175,6 +176,28 @@
         custom_fragment         => template('foreman/_assets.conf.erb', 'foreman/_ssl_virt_host_include.erb',
                                             'foreman/_suburi.conf.erb'),
       }
+
+      apache::vhost { 'foreman-ssl':
```

Comment:
On second thought, you can configure everything without needing a template. Check out how we do the Capsule reverse proxy: https://github.com/Katello/puppet-capsule/blob/master/manifests/reverse_proxy.pp#L13